pkg/proxy/ipvs/proxier.go

```diff
@@ -975,6 +975,9 @@ func (proxier *Proxier) OnEndpointsSynced() {
 	proxier.syncProxyRules()
 }
 
+// EntryInvalidErr indiates if an ipset entry is invalid or not
+const EntryInvalidErr = "error adding entry %s to ipset %s since entry is invalid"
```

Comment:
Can you reduce the error down to maybe "error adding entry %s to ipset %s". The part "since entry is invalid" That is a bit redundant.

Reply:
Yeah, it's cleaner. It's fixed now, thanks!